site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -305,17 +305,17 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.extensions.logLevel`                             | Agones Extensions Log level. Log only entries with that severity and above                                                                                                                                                                                                                                       | `info`                                    |
 | `agones.extensions.persistentLogs`                       | Store Agones extensions logs in a temporary volume attached to a container for debugging                                                                                                                                                                                                                         | `true`                                    |
 | `agones.extensions.persistentLogsSizeLimitMB`            | Maximum total size of all Agones container logs in MB                                                                                                                                                                                                                                                            | `10000`                                   |
-| `agones.extensions.disableSecret`                        | Disables the creation of any allocator secrets. If true, you MUST provide the `{agones.releaseName}-cert` secrets before installation.                                                                                                                                                                           | `false`                                   |
+| `agones.extensions.disableSecret`                        | Disables the creation of any allocator secrets. If not set, this parameter falls back to the value of `.Values.agones.controller.disableSecret`. You MUST provide the `{agones.releaseName}-cert` secrets before installation if this is set to `true`.                                                                                                                                                                           | `false`                                   |
```

Comment:
Can we please make a copy of this table with the changes and `{{ feature }}` shortcode it please.

Reply:
The feature shortcode has been included in both the Controller and Extensions. If I remember correctly, you mentioned that this shortcode is only for the new features.🤔